pkg/providers/v1/aws.go

```diff
@@ -3013,7 +3013,8 @@ func (c *Cloud) UpdateLoadBalancer(ctx context.Context, clusterName string, serv
 
 	err = c.ensureLoadBalancerInstances(aws.StringValue(lb.LoadBalancerName), lb.Instances, instances)
 	if err != nil {
-		return nil
+		klog.Warningf("Error registering/deregistring instances with the load balancer: %q", err)
+		return err
```

Comment:
this would cause an event and users can now know about the failures correct? If so , can we update the release notes about the same on change is behavior.

Reply:
That event won't happen before correct as we are not returning error. After this change the event will be shown during errors which would help in debugging correct?

Reply:
Yes, but that shouldn't be a event which introduces a "behavior" change. It should have been the expected behavior from the start which was not handled. I can call it out as a. bug in the release notes

Reply:
yes, that's correct. It's a bug which we are fixing and it would be nice to mention it in the release notes about the fixing it.

Reply:
Updated